scheduler/src/cook/mesos/api.clj

```diff
@@ -2028,7 +2027,7 @@
                    [false {::error (str "You are not authorized to list jobs for " user)}]
 
                    (or (< 168 since-hours-ago)
-                       (> 0 since-hours-ago))
+                       (neg? since-hours-ago))
                    [false {::error (str "since-hours-ago must be between 0 and 168 (7 days)")}]
```

Comment:
We should add "inclusive" to this error message.

Reply:
Used `interval [0, 168]` in the error message